### ISSUE TITLE
Sleeping Dogs: Definitive Edition (307690)

### ISSUE THREAD

**Issue by Xeers** [ValveSoftware/Proton#872]
Sleeping Dogs: Definitive Edition

Proton 3.7-3

Nvidia Geforce GTX 950
CPU AMD FX
https://gist.github.com/Xeers/219b4b3e5eb10c6ec225d7cc8c61be44

Pastebin:
https://paste.drhack.net/?f94ab50f168ef755#FmlLdO43duTgaRI1h2gkBRGCjKBUhXzsSTleKdvHQiQ=
Window with black screen

This does not work:
PROTON_USE_WINED3D11

With Proton 3.7-4-Beta the window of the game is left blank, the music of the game is heard and the arrow of the mouse is seen but then the game closes.

Pastebin:
https://paste.drhack.net/?696a8c7e04d92230#1b2Dj/nyDYZpp6twwkPF/viQFYSG18/tzsdCgljGkVc=

**Comment by HonkingGoose** [comment]
#596 There is some discussion on here too about the Definitive Edition.

**Comment by kisak-valve** [comment]
Closing as a duplicate of #596. Please at least try to search for open issues before making new ones.

**Comment by Xeers** [comment]
They are two different editions

**Comment by HonkingGoose** [comment]
### For reference:

From the [Sleeping Dogs: Definitive Edition Steam Store page](https://store.steampowered.com/app/307690/):
>The Definitive Edition of the critically acclaimed, award winning open-world action adventure, **reworked, rebuilt and re-mastered** for the new generation. 

So it's a different edition all together from the original Sleeping Dogs.

**Comment by mimattr** [comment]
## System Information
- GPU: **GTX 970**
- Driver/LLVM version: **NVIDIA 396.54**
- Kernel version: **4.4.0-135-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **3.7-7 Beta**

## I confirm:
- [ ] that pressing the `Play` button in the Steam client is sufficient.
- [x] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options)
      are necessary to run the game:
  1. **Configuration:** Add the following ["Launch" option](https://support.steampowered.com/kb_article.php?ref=1040-JWMT-2947) to **Sleeping Dogs: Definitive Edition** within Steam:
_[DXVK_CONFIG_FILE](https://github.com/doitsujin/dxvk/wiki/Configuration)="/path/to/[dxvk_sleepingdogs.conf](https://gist.github.com/mimattr/f7d096853193a5310481e4a53465a3d9)" %command%_
- [x] that no workarounds other than the mentioned ones are necessary.

## Issues
- [ ] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [x] Although I consider the gaming experience equal to Windows there are
      remaining issues:
  1. **Display:** Default `Fullscreen` mode not working as expected, all settings result in an [application window](https://gist.github.com/mimattr/38f5fd3a7e43b3e68356e4bef51b7a86#file-steam-307690-log-L5537)

**Comment by barni2000** [comment]
It works with **vd=1280x720** wine verb, after the game starts can be set **fullscreen mode** and resultion from game settings. The game needs the **dxvk** with **wined3d** it cannot start.

It is frequently crashing at some places in the game.

**Comment by Simmonz** [comment]
The game launches to a white screen for me.

[steam-307690.log](https://github.com/ValveSoftware/Proton/files/2913188/steam-307690.log)

Distro: Manjaro

GPU: RX 580 8GB

Driver/LLVM version: Mesa 18.3.3/7.0.1

Kernel version: 4.20.11-1-MANJARO

Proton version: 3.16-4

https://gist.github.com/Simmonz/7a6eb7ee79fcb5deee2a12633f5d8a58

**Comment by barni2000** [comment]
@Simmonz you need virtual desktop.

**Comment by ziabice** [comment]
Tested with Proton 4.2-1: game starts and shows only a white screen.

**Comment by lucifertdark** [comment]
barni2000 has the answer above, set a virtual desktop & the game runs very nicely.

**Comment by ziabice** [comment]
@lucifertdark yep, I'm aware, but this is Proton, not WINE, I don't have to use tricks to run games, that's why this game is not whitelisted and I'm reporting the test I did with Proton 4.2

**Comment by howels** [comment]
Works with ```protontricks 307690 vd=3840x1600``` but some crashes and poor controller compatibility.

**Comment by 5DavisSt** [comment]
@ziabice : Do you have a file called "DisplaySettings.xml" in the "data" folder of Sleeping Dogs? If you don't have it, follow the instructions provided by [this link](https://steamcommunity.com/app/307690/discussions/0/1700542332318710843/#c1700542332319844408). I never used protontricks, but I can play the game.

**Comment by ziabice** [comment]
@5DavisSt Yep, I have the file, thanks for the tip. I'll check this weekend and report back!

**Comment by frysztak** [comment]
About the crashing, it always happens in the mission Payback. Either somewhere in the middle or at the very end, when you have to evade the police. I took a look at the logs but I'm having hard time finding anything concrete. This stands out the most:
```
230029.322:0027:003a:err:seh:call_stack_handlers invalid frame 108738 (0x2bd2000-0x2d50000)
230029.322:0027:003a:err:seh:NtRaiseException Exception frame is not in stack limits => unable to dispatch exception.
```

Here's the zip file containing log, .dmp file and a savegame, if anyone wanted to reproduce this issue. [sleepy_dogs.zip](https://github.com/ValveSoftware/Proton/files/3138969/sleepy_dogs.zip)

**Comment by neuromancer** [comment]
`protontricks 307690 vd=X` *where X is your desktop resolution works fine for me. The Steam controller seems to work correctly, when you select the most popular configuration.

**Comment by Leopard1907** [comment]
Workaround for white screen ; `protontricks 307690 winecfg` and setting virtual desktop to match your desktop res works. 

However game is pretty inconsistent about succesfully loading into game. It can only success at one tries out of 3.

I grabbed logs from both situations. Disabled esync and fsync to make sure they are not the culprits.

[steam-307690-didn't work.log](https://github.com/ValveSoftware/Proton/files/3907702/steam-307690-didn.t.work.log)
[steam-307690-worked.log](https://github.com/ValveSoftware/Proton/files/3907703/steam-307690-worked.log)

**Comment by Guy1524** [comment]
Testing with a fresh prefix and 4.11-9, I am unable to reproduce the inconsistent issue mentioned by @Leopard1907

**Comment by mimattr** [comment]
As others have mentioned the application launches in "Windowed" mode by default (same behaviour as Windows 10), changing to "Fullscreen" within the application causes a white screen using **DXVK**:

![Fullscreen Screenshot 1](https://i.imgur.com/KxYmjAU.png)
![Fullscreen Screenshot 2](https://i.imgur.com/MFjgP0i.png)

Using **WineD3D** results in a black screen and fails to launch:

![WineD3D Screenshot](https://i.imgur.com/ocNM5yl.png)

This may be a Wine (DXGI) issue but @doitsujin should be able to confirm.

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 440.44**
- Kernel version: **5.0.0-37-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **4.11-11**

## I confirm:
- [x] that I have checked whether there are updates for my system available.

[steam-307690.ubuntu1804_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023419/steam-307690.ubuntu1804_dxvk15.log) (fullscreen issue)
[sdhdship_d3d11.ubuntu1804_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023442/sdhdship_d3d11.ubuntu1804_dxvk15.log) (fullscreen issue)
[SDHDShip_d3d11.windows10_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023443/SDHDShip_d3d11.windows10_dxvk15.log) (works as expected)
[SDHDShip_dxgi.windows10_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023444/SDHDShip_dxgi.windows10_dxvk15.log) (works as expected)
[SDHDShip.windows10_dx11.trace](https://mega.nz/#!VhxBRQqK!P_jWERRtANDvTuTa3GLOzq0eMeCIk4QJ_olkzwukdh0) (works as expected)
[sdhdship.ubuntu1804_dx11.trace](https://mega.nz/#!c153TaIA!PLTx6G3gHvw2Gz5KEBd_sIEZJ4Q-VdJUR6Mb_pl1Yvs) (fullscreen issue)

## Reproduction
1. Select the title within your `Library` using the Steam client and press the `Play` button
1. Select `Options` > `Display` > `Fullscreen` > `On` from the main menu

**Comment by mimattr** [comment]
As others have mentioned the application launches in "Windowed" mode by default (same behaviour as Windows 10), changing to "Fullscreen" within the application causes a white screen using **DXVK**:

![Fullscreen Screenshot 1](https://i.imgur.com/KxYmjAU.png)
![Fullscreen Screenshot 2](https://i.imgur.com/MFjgP0i.png)

Using **WineD3D** results in a black screen and fails to launch:

![WineD3D Screenshot](https://i.imgur.com/ocNM5yl.png)

This may be a Wine (DXGI) issue but @doitsujin should be able to confirm.

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 440.44**
- Kernel version: **5.0.0-37-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **4.11-11**

## I confirm:
- [x] that I have checked whether there are updates for my system available.
- [steam-307690.ubuntu1804_proton411-11.log](https://github.com/ValveSoftware/Proton/files/4024006/steam-307690.ubuntu1804_proton411-11.log) (`Fullscreen` issue)
- [sdhdship_d3d11.ubuntu1804_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023442/sdhdship_d3d11.ubuntu1804_dxvk15.log) (`Fullscreen` issue)
- [sdhdship.ubuntu1804_dx11.trace](https://mega.nz/#!c153TaIA!PLTx6G3gHvw2Gz5KEBd_sIEZJ4Q-VdJUR6Mb_pl1Yvs) (`Fullscreen` issue)
- [SDHDShip_d3d11.windows10_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023443/SDHDShip_d3d11.windows10_dxvk15.log) (works as expected)
- [SDHDShip_dxgi.windows10_dxvk15.log](https://github.com/ValveSoftware/Proton/files/4023444/SDHDShip_dxgi.windows10_dxvk15.log) (works as expected)
- [SDHDShip.windows10_dx11.trace](https://mega.nz/#!VhxBRQqK!P_jWERRtANDvTuTa3GLOzq0eMeCIk4QJ_olkzwukdh0) (works as expected)

## Reproduction
1. Select the title within your `Library` using the Steam client and press the `Play` button
1. Select `Options` > `Display` > `Fullscreen` > `On` from the main menu

**Comment by Vladislavuz** [comment]
The game doesn't work in fullscreen mode. Before starting, set the fullscreen mode value to 0 in the file SleepingDogsDefinitiveEdition/data/DisplaySettings.xml

**Comment by Vladislavuz** [comment]
On SteamOS, it is not possible to launch due to the inability to display additional Windows before the first launch.

**Comment by jeselnik** [comment]
For those on the RADV driver prepending [gamescope](https://github.com/Plagman/gamescope) to your launch options can fix the white screen problem a bit more elegantly. Example: `gamescope -w 1920 -h 1080 -f -- %command%`. I've found this to also be necessary with tiling window managers as they attempt to tile the initial game window which messes up output even when it goes fullscreen

**Comment by ipr** [comment]
As others have mentioned it does not work in fullscreen (white screen) but it launches directly to fullscreen now (not windowed as someone said). The configuration file mentioned above is missing by default as well. So adding DisplaySettings.xml is necessary to get into main menu.

**Comment by ahjolinna** [comment]
I just tried this on proton-expertimental [bleeding-edge] and the full screen issue is still relevant

I can give my logs if needed



-------

My System Specs:


```
             .;ldkO0000Okdl;.                ahjolinna@localhost
         .;d00xl:^''''''^:ok00d;.            OS: openSUSE 20211210
       .d00l'                'o00d.          Kernel: x86_64 Linux 5.15.6-1-default
     .d0K^'  Okxoc;:,.          ^O0d.        Uptime: 20h 52m
    .OVVAK0kOKKKKKKKKKKOxo:,      lKO.       Packages: 2853
   ,0VVAKKKKKKKKKKKKK0P^,,,^dx:    ;00,      Shell: bash 5.1.12
  .OVVAKKKKKKKKKKKKKk'.oOPPb.'0k.   cKO.     Resolution: 3840x2160
  :KVAKKKKKKKKKKKKKK: kKx..dd lKd   'OK:     DE: KDE 5.89.0 / Plasma 5.23.80
  lKlKKKKKKKKKOx0KKKd ^0KKKO' kKKc   lKl     WM: KWin
  lKlKKKKKKKKKK;.;oOKx,..^..;kKKK0.  lKl     GTK Theme: Materia-dark [GTK2/3]
  :KAlKKKKKKKKK0o;...^cdxxOK0O/^^'  .0K:     Icon Theme: Papirus
   kKAVKKKKKKKKKKKK0x;,,......,;od  lKP      Disk: 17T / 25T (68%)
   '0KAVKKKKKKKKKKKKKKKKKK00KKOo^  c00'      CPU: AMD Ryzen 7 5700G with Radeon Graphics @ 16x 4.3GHz
    'kKAVOxddxkOO00000Okxoc;''   .dKV'       GPU: NVIDIA GeForce RTX 3060 Ti
      l0Ko.                    .c00l'        RAM: 5322MiB / 15771MiB
       'l0Kk:.              .;xK0l'         
          'lkK0xc;:,,,,:;odO0kl'            
              '^:ldxkkkkxdl:^'

```

**Comment by Qemics** [comment]
I enabled virtual desktop through lutris and set the resolution to 1280x800 and nothing happens.

[lutris debug log](https://dpaste.org/KbB8)

**Comment by GamertechAU** [comment]
Attempting to load this game on Fedora/Wayland brings up a light blue window. I can hear the loading videos looping over and over but can't see anything other than the blue screen.

Added DisplaySettings, set virtual desktop, Gamescope/Lutris/Bottles etc, nothing I do changes it. I can't see anyone else mentioning the blue, only white screens, so I think it might have something to do with my Radeon 7900 XT.

Mesa 23.x which adds more support for the card isn't released stable yet so it's not fully 'recognised' by the distro, but this is the only game that seems to have this issue.

Edit: Mesa 23.0.1 doesn't change anything with SD:DE

Edit 2: On first run SD:DE installs msvcrun2010, but it seems it's not 100% reliable. Deleting the prefix and re-running a few times and msvcrun installed correctly and the game runs.

**Comment by weefunker** [comment]
> The game doesn't work in fullscreen mode. Before starting, set the fullscreen mode value to 0 in the file SleepingDogsDefinitiveEdition/data/DisplaySettings.xml

This solved it for me

**Comment by Tiagoquix** [comment]
As of 2024-06-07, Proton Experimental changelog lists "Sleeping Dogs: Definitive Edition" as "now playable".

What was changed? I can't see to find anything regarding this specific line in the changelog.

**Comment by kisak-valve** [comment]
Sleeping dogs definitive edition not launching then stoping fix in Kali Linux

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7947.
@vidakpop posted on 2024-07-29T10:09:05:


I have used proton experimental but the launch was flickering then stoping.I fixed the issue by adding this code on the launch options 

`PROTON_USE_WINED3D=1 %command%`
***
@Kuratius commented on 2024-07-29T11:59:02:

https://github.com/ValveSoftware/Proton/issues/872

Try to avoid multiple tracking issues for the same piece of software.

**Comment by GalecO1987** [comment]
> As of 2024-06-07, Proton Experimental changelog lists "Sleeping Dogs: Definitive Edition" as "now playable".
> 
> What was changed? I can't see to find anything regarding this specific line in the changelog.

In the latest stable version of Proton there is still a problem with a white screen when launching the game.
I can confirm that this game now works perfectly fine with Experimental, no issues.

**Comment by Tiagoquix** [comment]
@GalecO1987 This game works fine using Proton-GE (it has a protontricks system, which applies automatic game fixes). See https://github.com/Open-Wine-Components/umu-protonfixes/blob/master/gamefixes-steam/307690.py for details.

**Comment by ranplayer** [comment]
folks, the fullscreen mode is working with Proton 9.0-4, not fully working with 10.0-3 and working with Experimental 2025-11-26 (after some resolution changes)

Proton 10.0-3 logs: [steam-307690.tar.gz](https://github.com/user-attachments/files/23808723/steam-307690.tar.gz)

Picture from 10.0-3:

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/6c9dee0c-a22a-4dba-b342-e7f7f8a25bc9" />

Specs:
- OS: Arch
- Kernel: 6.12.59
- DE: Plasma 6.5.3 (wayland)
- GPU: RX 6700 XT (Mesa 25.2.7)
- CPU: Ryzen 7 5700G

**Comment by kisak-valve** [comment]
Hello @ranplayer, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by ranplayer** [comment]
done @kisak-valve

**Comment by LeoKeidran** [comment]
Unsure if this is the appropriate issue to post this to, so apologies in advance if this is the wrong place.

Specs:
- OS: Linux Mint 22.3 - Cinnamon 64-bit
- Kernel: 6.17.0-29-generic
- DE: Cinnamon 6.6.7 (X11)
- GPU: RX 7900 GRE (Mesa 25.2.8-0ubuntu0.24.04.1)
- CPU: AMD Ryzen 5 7600X

When using Proton version 10 or newer, inputs stick if I run LiveSplit in the same Proton instance. Running the game with Proton 9 however does not have this issue. But this has some other issues that make using this Proton version unappealing.

I can see there is an issue for sticky inputs, but this doesn't quite match that issue: #5294. Killing ibus does not resolve the issue. The issue is only present when LiveSplit is running in the same instance.

**Comment by codeweaverwill** [comment]
> When using Proton version 10 or newer, inputs stick if I run LiveSplit in the same Proton instance. Running the game with Proton 9 however does not have this issue. But this has some other issues that make using this Proton version unappealing.

Could you provide repro steps for how you're using LiveSplit in this case? Would love to try to reproduce this.

**Comment by LeoKeidran** [comment]
Of course!
First, boot SD: DE with Proton 10 or newer. Once the game has started, use protontricks to launch LiveSplit, eg: `protontricks-launch --no-bwrap --appid 307690 ~/LiveSplit/LiveSplit.exe` (to wherever your LiveSplit exe is). In game, load a save to enter free-roam and run around. The way I test this is to hold all four direction inputs (WASD) and then let go. The character will then continue to run even after you let go of the key.
Doing these exact same steps in Proton 9 doesn't have this 'sticky input' issue.

**Comment by LeoKeidran** [comment]
Small follow up, I tested this with the latest Proton version, 11.0-2, just to confirm that the sticky input issue is still present. I've also grabbed the logs as well, in case that is helpful.

I also spent a bit of time just checking if any changes to Environment Variables would make a difference, but no. I have patches applied to the game, but checked and saw the behaviour was the same without an ASI injector.

[steam-307690.log](https://github.com/user-attachments/files/31888413/steam-307690.log)
